### ISSUE TITLE
hope The Range.Copy method will only copy certain ExcelRangeCopyOptionFlag, and currently excludes certain ExcelRangeCopyOptionFlags

### ISSUE THREAD

**Issue by minren118** [EPPlusSoftware/EPPlus#1656]
In some cases, the Copy method is used to copy only certain ExcelRangeCopyOptionFlagof , instead of copying all types and excluding certain ExcelRangeCopyOptionFlags. For example, I just want to copy the formula, I don't need to copy anything else.

Like the VBA interface, its special paste is only for a certain type of paste，It's single-choice

![image](https://github.com/user-attachments/assets/0dd595e3-d1e2-4323-9284-a83e201cac2b)

Finally, there is a special case, that is, the selection is a cell, can not be copied to multiple cells, in the VBA interface, this can be very flexible, and even can be copied to multiple discontinuous Ranges.

The current status of EPPLUS is to copy to the target cell Range, and if the source region is a cell, the copied result is also an top left cell in this continuous region

```
Sub testCopyMultiRange()

Dim rng As Range

Set rng = Range("G10")
rng.Copy Range("J15:J20,G11:G20")

End Sub
```

**Comment by JanKallman** [comment]
Yes, I assume we could add a new overload with a new enum with "include flags" instead.
A "copy fill" function or similar is on our enhancement list, see #1271.
I'll add this issue you our enhancement project.

**Comment by nltheme** [comment]
maybe this solution will solve your problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by JanKallman** [comment]
Add a CopyValues function.

**Comment by OssianEPPlus** [comment]
A fix for this was added in Epplus 8.1

Future updates to paste-special tracked in #1974
